pymodbus/transport/transport.py

```diff
@@ -88,7 +88,7 @@ class CommParams:
     comm_name: str | None = None
     comm_type: CommType | None = None
     reconnect_delay: float | None = None
-    reconnect_delay_max: float | None = None
+    reconnect_delay_max: float = 0.0
```

Comment:
Actually there is a hidden problem here, "reconnect_delay_max" is defined a milliseconds and is an integer.

It seems asyncio.py does still use seconds, so I have to look at code before I approve this one.